### PR TITLE
Device orientation and motion fixes related to Events

### PR DESCRIPTION
**Pull request by hamishwillee**
Device orientation and motion fixes for #2245

Bit of a work in progress, because the state of the device orientation APIs seems a little "flakey".

### CODE REVIEW

**Comment by hamishwillee** [comment]
@chrisdavidmills Who would be the right person to talk to about device orientation/motion events and APIs? It all seems a bit flakey:
- [DeviceMotionEvent](https://developer.mozilla.org/en-US/docs/Web/API/DeviceMotionEvent) 
  -  has a warning "Currently, Firefox and Chrome do not handle the coordinates the same way.". It would be good to clarify that.
  - propose remove experimental as everyone has implemented it, even though there are obviously some differences.
- [DeviceOrientationEvent](https://developer.mozilla.org/en-US/docs/Web/API/DeviceOrientationEvent)
  - also marked experimental.
  - has an `absolute` field indicating that values are relative to earth frame or not. Default from [chrome 50](https://developers.google.com/web/updates/2016/03/device-orientation-changes) appears to be to provide relative values and suggest that users try [Window.ondeviceorientationabsolute](https://developer.mozilla.org/en-US/docs/Web/API/Window/ondeviceorientationabsolute) (not supported by FF).
  - BUT not clear what relative actually means - relative to what? Degrees since last event?
- [Window.orientationchange_event](https://developer.mozilla.org/en-US/docs/Web/API/Window/orientationchange_event) is deprecated. Can you confirm that `window.screen.orientation` and `screen.orientation.onchange` are the right way to do this now?

**Comment by hamishwillee** [comment]
Note, not going to merge this do to conflicts. But keeping it alive for discussion above.

**Comment by chrisdavidmills** [comment]
> @chrisdavidmills Who would be the right person to talk to about device orientation/motion events and APIs? It all seems a bit flakey:

You mean as in, an engineer to ask clarifying questions to?

**Comment by hamishwillee** [comment]
@chrisdavidmills 

> You mean as in, an engineer to ask clarifying questions to?

Yes :-). Someone who will also know what Chrome is doing/differences.

**Comment by chrisdavidmills** [comment]
> @chrisdavidmills
> 
> > You mean as in, an engineer to ask clarifying questions to?
> 
> Yes :-). Someone who will also know what Chrome is doing/differences.
> Note, I can experiment. But I'm not sure how much time you want spent on this.

I'm not sure. @jpmedley might be able to suggest someone on the Chrome side.

File: files/en-us/web/guide/events/orientation_and_motion_data_explained/index.html [hunk 1]
```diff
@@ -42,7 +42,10 @@ <h3 id="Device_coordinate_frame">Device coordinate frame</h3>
  <li>The <strong>z</strong> axis is perpendicular to the screen or keyboard, and is positive extending outward from the screen.</li>
 </ul>
 
-<div class="note"><strong>Note:</strong> On a phone or tablet, the orientation of the device is always considered in relation to the standard orientation of the screen; this is the "portrait" orientation on most devices. On a laptop computer, the orientation is considered in relation to the keyboard. If you want to detect changes in device orientation in order to compensate, you can use the <code>orientationchange</code> event.</div>
+<div class="notecard note">
+  <h4>Note</h4>
+  <p>On a phone or tablet, the orientation of the device is always considered in relation to the standard orientation of the screen; this is the "portrait" orientation on most devices. On a laptop computer, the orientation is considered in relation to the keyboard. If you want to detect changes in device orientation in order to compensate, you can use the <code>orientationchange</code> event.</p>
```
Comment (jpmedley) [inline files/en-us/web/guide/events/orientation_and_motion_data_explained/index.html:47]:
```suggestion
  <p>On a phone or tablet, the orientation of the device is always considered in relation to the standard orientation of the screen; this is the "portrait" orientation on most devices. On a laptop computer, the orientation is considered in relation to the keyboard. If you want to detect changes in device orientation in order to compensate, you can use the {{event("orientationchange")}} event.</p>
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: files/en-us/web/guide/events/orientation_and_motion_data_explained/index.html (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/web/guide/events/orientation_and_motion_data_explained/index.html:47]: FYI, I'm pretty sure that would not work - the macro is a bit broken and points to the wrong place. Events now live under the object that fires them or the global event thingy in some cases.

**Pull request closed without merging**